### PR TITLE
ci: regenerate package lock file

### PR DESCRIPTION
**Pull request by mrrajamanickam-coveo**
https://coveord.atlassian.net/browse/KIT-2429

Also 
* fix headless version in hosted pkg
* add a npm script to perform clean install

### CODE REVIEW

File: packages/atomic-hosted-page/src/components.d.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export namespace Components {
           * @param organizationId
           * @param env
          */
-        "getOrganizationEndpoints": (organizationId: string, env?: PlatformEnvironment) => Promise<{ platform: string; analytics: string; search: string; admin: string; }>;
```
Comment (mrrajamanickam-coveo) [inline packages/atomic-hosted-page/src/components.d.ts:16]: Hmm.. did this get missed in a previous PR? 🤔

File: package.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "private": true,
   "scripts": {
     "postinstall": "husky install && patch-package && nx run-many --target=install:isolated",
+    "clean:install": "rm -f package-lock.json && rimraf -rf **/node_modules && npm install",
```
Comment (louis-bompart) [inline package.json:6]: `rm` doesn't behave the same on windows, use `rimraf`, or remove `-f`

File: packages/atomic-hosted-page/src/components.d.ts (hunk 1 above)
Comment (mrrajamanickam-coveo) [reply packages/atomic-hosted-page/src/components.d.ts:16]: npm run build fails locally with 
```
  [59:08.5]  build failed in 133 ms


        >  NX   ERROR: Something went wrong in run-commands - Command failed: node --max_old_space_size=6144 ../../node_modules/@stencil/core/bin/stencil build
```

**Review by olamothe — APPROVED** [review]

**Comment by mrrajamanickam-coveo** [comment]
All checks passing except [Report bundle size](https://github.com/coveo/ui-kit/actions/runs/4802283107/jobs/8545552883?pr=2846) as it does `npm ci` on master to compare I guess - Since master is broken looks like I have to merge this PR anyways.

**Pull request merged**